<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>date::SettingsDialog</class>
 <widget class="QDialog" name="date::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date plugin settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="format_group">
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="int_type_button">
        <property name="text">
         <string>system</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="int_type_box"/>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="str_type_button">
        <property name="text">
         <string>custom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="str_type_box">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string notr="true">dd.MM.yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">MM/dd/yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">M/d/yyyy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">yy/MM/dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">yyyy-MM-dd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">d-MMM-yy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">dddd, MMMM d</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="font_group">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="clock_font_button">
        <property name="text">
         <string>use clock font</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="custom_font_button">
        <property name="text">
         <string>custom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="choose_font_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="font_autosize">
        <property name="text">
         <string>auto calculate font size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>date::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>date::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>int_type_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>int_type_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>str_type_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>str_type_box</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_font_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>choose_font_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
